src/components/app.tsx

```diff
@@ -57,22 +57,20 @@ class AppBase extends React.PureComponent<any, {}> {
     );
   }
 
-  private update(props: any) {
+  private update(props: VoyagerAppProps) {
     const { data } = props;
-    if (VegaLiteData.isUrlData) {
+    if (isUrlData(data)) {
```

Comment:
@tafsiri FYI, type guard are methods so you have to call them too.  

Also, i found basarat.gitbooks.io/typescript/ useful when I started working on TS. :)